### ISSUE TITLE
get_player mock shouldn't fail if auth dialog wasn't shown

### ISSUE THREAD

**Issue by S0Close** [indiesoftby/defold-yagames#17]
Yandex SDK allows to fetch player even if the user is not authorized. In this case only unique_id is available. For example, you can use cloud saves even without signing up on yandex games.
`
if not M._auth then
        M.send(cb_id, "FetchError: Unauthorized")
`
https://github.com/indiesoftby/defold-yagames/blob/4c925e4e2e96eebc36af3dbb68ef37fc10036325/yagames/helpers/mock.lua#L221
This check makes it mandatory to show auth dialog before calling get_player even if you don't need authorization

**Comment by aglitchman** [comment]
> For example, you can use cloud saves even without signing up on yandex games.

Yes, you are absolutely right. A long time ago, when this extension was created, Yandex didn't have mode "lite" yet.
I'll take a look!
